week-1/mandatory/2-classes-db/task.md

```diff
@@ -7,7 +7,25 @@ Below you will find a set of tasks for you to complete to consolidate and extend
 To submit this homework write the correct commands for each question here:
 
 ```sql
-
+1.select * from rooms where rate >100;
+2. select * from reservations where checkin_date between '01-09-2020' a
+nd '30-09-2020' and checkout_date-checkin_date > 3;
+3.select * from customers where name like 'M%';
+4. insert into room_types (room_type,def_rate) values ('PENTHOUSE',185.
+0);
+5. insert into rooms (room_no,room_type,rate) values(501,'PENTHOUSE',18
+5),(502,'PENTHOUSE',185);
+6. insert into rooms (room_no,room_type,rate) values(503,'PREMIUM PLUS'
+,143);
+7. select count(room_no) from reservations where checkin_date between '
+01-08-2020' and '30-08-2020' and checkout_date-checkin_date >0;
```

Comment:
> how many rooms were occupied any time during the previous month

you have to distinct on the room_no, otherwise, reservations on the same room will be taken into as many times as many reservations it has.
`count(distinct room_no)`

Reply:
I'm not sure why this condition is necessary: `and checkout_date-checkin_date >0`